packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1879,6 +1888,11 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
       withComposing: !widget.readOnly,
     );
   }
+
+  @override
+  void connectionClosed() {
+    _closeConnectionReceived();
```

Comment:
I'm curious why not inline this method here?

Reply:
done. I also carried the method. I want all close/open related methods to be close, to group a subject area.